pandas/core/arrays/datetimes.py

```diff
@@ -361,7 +372,7 @@ def __init__(self, values, dtype=_NS_DTYPE, freq=None, copy=False):
                 "ndarray, or Series or Index containing one of those."
             )
             raise ValueError(msg.format(type(values).__name__))
-        if values.ndim != 1:
+        if values.ndim not in [1, 2]:
```

Comment:
Why was this changed? Seems to conflict with the error message directly below

Reply:
Applicable in a few places

Reply:
Yah, this is kludge-adjacent.  We don't _really_ support 2D, so don't to tell users its OK.